spec/ar2dto/record_to_dto/attributes_spec.rb

```diff
@@ -20,7 +21,9 @@
       it { is_expected.to be_a(UserDTO) }
 
       it "doesn't expose ActiveRecord's methods" do
-        expect(subject).not_to respond_to(:update)
+        expect(subject).to_not respond_to(:update)
+        expect(subject).to_not respond_to(:pending?)
```

Comment:
To make this explicit. I first thought that these could be ActiveModel auto generated methods and we may want to have them in DTOs, but then saw that they are from ActiveRecord, so probably not necessary.